gtest/src/manager.rs

```diff
@@ -461,12 +464,12 @@ impl ExtManager {
         }
     }
 
-    pub(crate) fn read_state_bytes_using_wasm(
+    pub(crate) fn read_state_bytes_using_wasm<E: Encode + TypeInfo + 'static>(
         &mut self,
         program_id: &ProgramId,
         fn_name: &str,
         wasm: Vec<u8>,
-        argument: Option<Vec<u8>>,
+        args: Option<E>,
```

Comment:
this fn is not codec-compatible, avoid it here or introduce another

Reply:
But `E: Encode` or what do you mean?